scripts/voqutil

```diff
@@ -0,0 +1,337 @@
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_SPEED)))
+
+        return table
+
+    @multi_asic_util.run_on_multi_asic
+    def get_systemport_status(self):
+        self.appl_db_keys_system_port = appl_db_keys_get_system_port(self.db, None)
+        if self.appl_db_keys_system_port:
+            self.table += self.generate_systemport_status()
+
+# ========================== voq system neigh status logic ==========================
+
+header_system_neigh = ['System Port Interface', 'Neighbor', 'MAC', 'Encap Index']
+
+class SystemNeighStatus(object):
+
+    def __init__(self, namespace_option, display_option):
+        self.db = None
+        self.config_db = None
+        self.table = []
+        self.multi_asic = multi_asic_util.MultiAsic(
+            display_option, namespace_option)
+
+    def display_systemneigh_status(self):
+
+        self.get_systemneigh_status()
+
+        # Sorting and tabulating the result table.
+        sorted_table = natsorted(self.table)
+        print(tabulate(sorted_table, header_system_neigh, tablefmt="simple", stralign='right'))
+
+    def generate_systemneigh_status(self):
+        """
+            Generate system neigh status output
+        """
+
+        i = {}
+        table = []
+        key = []
+
+        #
+        # Iterate through all the keys and append system neigh's associated info to
+        # the result table.
+        #
+        for i in self.chassis_app_db_keys_system_neigh:
+            key_tokens = re.split('\|', i)
+            nbr = key_tokens[-1].strip()
+            intf = '|'.join(key_tokens[1:-1])
+            key = '|'.join(key_tokens[1:])
+            table.append((intf, nbr,
+                            chassis_app_db_system_neigh_status_get(self.db, key, SYSTEM_NEIGH_MAC),
+                            chassis_app_db_system_neigh_status_get(self.db, key, SYSTEM_NEIGH_ENCAP_INDEX)))
+        return table
+
+    def get_systemneigh_status(self):
+        self.db = swsscommon.SonicV2Connector(use_unix_socket_path=False)
+        self.db.connect(self.db.CHASSIS_APP_DB, False)
+        self.chassis_app_db_keys_system_neigh = chassis_app_db_keys_get_system_neigh(self.db)
+        if self.chassis_app_db_keys_system_neigh:
+            self.table += self.generate_systemneigh_status()
+
+# ========================== VOQ system lag status logic ==========================
+
+header_system_lag = ['System Lag Name', 'Lag Id', 'Switch Id', 'Member System Ports']
+
+class SystemLagStatus(object):
+
+    def __init__(self, system_lag_name, namespace_option, display_option):
+        """
+        Class constructor method
+        :param self:
+        :param intf_name: string of system lag name
+        :return:
+        """
+        self.db = None
+        self.config_db = None
+        self.system_lag_name = system_lag_name
+        self.table = []
+        self.multi_asic = multi_asic_util.MultiAsic(
```

Comment:
Same as above, looks `self.multi_asic` like this is not used ?

Reply:
Fixed. Removed since this is not used.